workerpool/example/consumer.go

```diff
@@ -46,6 +61,7 @@ EXIT:
 			default:
 				panic(fmt.Sprintf("invalid type: %v", task))
 			}
+			c.completed++
```

Comment:
Moving this here felt right to me especially when I submit a PR for error counters and possibly a circuit breaker over max errors.

Reply:
Yeah, it would have probably been better named, `receivedWork` and then had a `completed` counter right where you put it.